lib/std/net.zig

```diff
@@ -731,7 +731,10 @@ pub fn getAddressList(allocator: mem.Allocator, name: []const u8, port: u16) !*A
         break :blk result;
     };
     const arena = result.arena.allocator();
-    errdefer result.arena.deinit();
+    errdefer {
+        var arena_copy = result.arena;
+        arena_copy.deinit();
+    }
```

Comment:
```suggestion
    errdefer result.deinit();
```

`AddressList.deinit` already does this with a comment explaining why.

Reply:
Lol, looks like we shot ourselves twice with the same footgun :)

Reply:
Done